src/http/native-http-backend.ts

```diff
@@ -93,6 +96,7 @@ export class NativeHttpConnectionD implements Connection {
                                 body: response.data,
                                 status: response.status,
                                 headers: new Headers(response.headers),
+                                url: req.url,
```

Comment:
Could we please use `response.url` here? This will help the library to correctly handle redirects.